content/blog/oidc-trust-relationships/index.md

```diff
@@ -71,11 +71,11 @@ In this demo, we are going to use Github Actions to retrieve Pulumi credentials
 3. Add a policy to allow OIDC and configure the sub and audience for your organization and repositories. In the demo, we are using:
 
 <!-- markdownlint-disable no-bare-urls -->
-* **Aud**: https://github.com/***organization***
+* **Aud**: urn:pulumi:org:***organization***
 
 * **Sub**: repo:***organization***/***repo***:*
 <!-- markdownlint-enable no-bare-urls -->
-4. Create a GitHub action. Here is a sample code. Make sure to substitute the `aud` claim with your organization's name in the `fetch pulumi token` step.
+4. Create a GitHub action. Here is a sample code. Make sure to substitute the `aud` claim with your organization's name in the `pulumi/auth-actions` parameters.
```

Comment:
Should this say :

4. Create a GitHub action. Here is a sample code. Make sure to substitute your organization in the `pulumi/auth-actions` organization parameter.